org.eclipse.xtext.common.types.eclipse.tests/META-INF/MANIFEST.MF

```diff
@@ -4,18 +4,18 @@ Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.xtext.common.types.eclipse.tests;singleton:=true
 Bundle-Version: 2.37.0.qualifier
 Bundle-Vendor: %providerName
-Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-RequiredExecutionEnvironment: JavaSE-17
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.xtext.common.types;bundle-version="2.37.0",
- org.eclipse.emf.ecore;bundle-version="2.26.0",
+ org.eclipse.emf.ecore;bundle-version="2.36.0",
  org.eclipse.emf.mwe2.runtime;bundle-version="2.20.0",
  org.eclipse.xtext;bundle-version="2.37.0",
- org.eclipse.core.resources;bundle-version="3.16.100",
- org.eclipse.jdt.core;bundle-version="3.29.0",
- org.eclipse.core.runtime;bundle-version="3.24.100",
- org.eclipse.jdt.launching;bundle-version="3.19.500",
+ org.eclipse.core.resources;bundle-version="3.20.100",
+ org.eclipse.jdt.core;bundle-version="3.37.0",
+ org.eclipse.core.runtime;bundle-version="3.31.0",
+ org.eclipse.jdt.launching;bundle-version="3.21.100",
  org.eclipse.xtext.common.types.ui;bundle-version="2.37.0",
- org.eclipse.emf.common;bundle-version="2.24.0",
+ org.eclipse.emf.common;bundle-version="2.30.0",
  org.eclipse.xtext.ui;bundle-version="2.37.0",
  org.eclipse.xtext.ui.shared;bundle-version="2.37.0",
  org.eclipse.ui.editors;bundle-version="3.14.300",
```

Comment:
Do you know whether there exists a utility that would update all require-bundle versions to match a selected target platform?

Reply:
Not that I know of, no.

Reply:
It would indeed be nice. E.g. API-Tools could suggest to use the current version from the TP as lower bound if the bundle has changed it's version.
Or if no internals are used at all API-Tools could even try to be smart and compute the lower bound from `@since` tags.